src/workspacefactory/wfactory_model.js

```diff
@@ -378,7 +378,7 @@ WorkspaceFactoryModel.prototype.addCustomTag = function(category, tag) {
  * @param {!Element} xml The XML to be saved.
  */
 WorkspaceFactoryModel.prototype.savePreloadXml = function(xml) {
```

Comment:
I have mild concern where this method is going.  If the workspace is already represented by input param `xml`, then what is this method doing? If it is only writing the actual file, maybe the responsibility lies in the app controller.